src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,91 @@
-import { useState } from 'react';
+import { FC, FormEvent, useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
-  // Increase the count after successful form submission
-  // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+type NewMovieProps = {
+  onAdd: (movie: Movie) => void
+};
+
+export const NewMovie: FC<NewMovieProps> = ({ onAdd }) => {
+  const [count, setCount] = useState(0);
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [imgUrl, setImgUrl] = useState('');
+  const [imdbUrl, setImdbUrl] = useState('');
+  const [imdbId, setImdbId] = useState('');
```

Comment:
You can use only 1 state for all fields.
Create const outside your component where you can save your `initialFormState` which will have all initial values 

```
const initialFormState = {
  title: '',
  ...
};
```

And find the way how to handle your changes to formState

```suggestion
  const [formState, setFormState] = useState(initialFromState);
```